examples/user.yaml

```
@@ -0,0 +1,28 @@
+types:
+  - name: id
+    id: true
+    inline: BIGSERIAL
+  - name: name
+    inline: VARCHAR(255)
+
+tables: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)